packages/backend/src/apps/toolbox/actions/if-then/index.ts

```diff
@@ -78,7 +78,12 @@ const action: IRawAction = {
       type: 'string' as const,
       label: 'FILE A BUG IF YOU SEE THIS',
 
-      hidden: true,
+      // Always hidden
+      hiddenIf: {
```

Comment:
nit: would this be the format we will follow for "always hidden" fields? if yes should we make this a constant somewhere?

Reply:
good idea. actually i think i got a better idea, I will see if I can improve the typing for this via discriminated unions.

My concern about makign this a constant is that people will forget the constant exists.